<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogVecFly</class>
 <widget class="QDialog" name="dialogVecFly">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialogVecFly</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction of travel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>How to draw the track</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Loc along track</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Averaging</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="forwardButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Forward (as digitized)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="reverseButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="overButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Track on top</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mixedButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Track mixed with terrain</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flyover controls look direction (azimuth)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSlider" name="locSlider">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>80</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="speedSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="avgSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doLook(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reverseButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doReverse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixedButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doMixed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doForward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>locSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doLoc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speedSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doSpeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>avgSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogVecFly</receiver>
   <slot>doAvg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doStart()</slot>
  <slot>doLook(bool)</slot>
  <slot>doForward()</slot>
  <slot>doReverse()</slot>
  <slot>doTop()</slot>
  <slot>doMixed()</slot>
  <slot>doLoc(int)</slot>
  <slot>doSpeed(int)</slot>
  <slot>doAvg(int)</slot>
  <slot>doLoc()</slot>
  <slot>doSpeed()</slot>
  <slot>doAvg()</slot>
 </slots>
</ui>
